packages/p2p/src/stores/general-store.js

```diff
@@ -433,7 +433,6 @@ export default class GeneralStore extends BaseStore {
     }
 
     onMount() {
-        this.setIsLoading(true);
```

Comment:
here we are removing `setIsLoading(true)` when p2p is mounting because now we are showing the Loader in index.tsx file above:
` () => <Loading is_fullscreen={false} />`